dts/bindings/sensor/hamamatsu,s11059.yaml

```diff
@@ -0,0 +1,42 @@
+# Copyright (c) 2022, Hiroki Tada
+# SPDX-License-Identifier: Apache-2.0
+
+description: |
+  Hamamatsu Photonics S11059 Color Sensor.  See datasheet at
+  https://datasheetspdf.com/pdf/1323325/Hamamatsu/S11059-02DT/1
+
+compatible: "hamamatsu,s11059"
+
+include: i2c-device.yaml
+
+properties:
+
+  gain:
+    type: int
+    description: |
+      Gain selection.
+      "1" is high gain mode and "0" is low gain mode.
+      This bit is selecting the photodiode area. 
+      The size ratio of high gain photodiode area 
+      and low gain photodiode area is 10 : 1. Therefore 
+      the gain ratio is 10 times from low to high.
+
+      This value is set as following:
+      0 = Low gain
+      1 = High gain
+      The default value is Low gain.
+    enum:
+      - 0
+      - 1
+
+  integration-time:
+    type: int
+    description: |
+      Integration time (unit is us).
+      By setting this value to the desired integration time, 
+      the Integration time setting and Manual timing register 
+      values are set automatically.
+
+      The default value is 546ms.
```

Comment:
Default values need justification. Does this correspond to the register reset value? It's also confusing to state the default in the devicetree binding, but set the default in the driver.

Reply:
This default value is set based on the initial value of the integration-time specified in the datasheet.
Also, I will remove the default value setting in the driver and set the value in the devicetree binding.

Reply:
Please note that it comes from the datasheet, and ideally where in the datasheet it comes from if you can in the description. This would be helpful to the next person that looks at this.

Reply:
I added a comment about it to the description.